docs/source/upgrading.rst

```diff
@@ -1,6 +1,40 @@
 Upgrading
 #########
 
+2.X to 3.X
+==========
+
+`acl` parameter is deprecated and a class decorated with `@resource`
+or `@service` becomes its own `route factory
+<http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#route-factories>`_ and
+it may take advantage of `Pyramid ACL
+<https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/security.html#assigning-acls-to-your-resource-objects>`_.
+
+Before:
+
+.. code-block:: python
+
+    @resource(path='/users')
```

Comment:
Here I think it would be relevant to have  `acl=[(Allow, Everyone, 'view')]` no?